src/libraries/System.ComponentModel.Annotations/tests/System.ComponentModel.Annotations.Tests.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <IncludeRemoteExecutor>true</IncludeRemoteExecutor>
     <TargetFrameworks>$(NetCoreAppCurrent);net48</TargetFrameworks>
-    <Nullable>disable</Nullable>
+    <Nullable>disable</Nullable> <!-- Disable nullable attributes as some tests depend on them not being present. -->
```

Comment:
(nit) is VS going to mess with this comment placement?

Reply:
Probably. :(

Reply:
Agreed. I wish VS didn't do it... especially when writing Tasks or lining up XML attributes. I'm sure that feedback is somewhere, I should probably go upvote it.